core/modules/field_ui/field_ui.admin.inc

```diff
@@ -45,26 +46,39 @@ function field_ui_fields_list() {
           $rows[$field_name]['class'] = $field['locked'] ? array('menu-disabled') : array('');
           $rows[$field_name]['data'][0] = $field['locked'] ? t('@field_name (Locked)', array('@field_name' => $field_name)) : $field_name;
           $module_name = $field_types[$field['type']]['module'];
-          $rows[$field_name]['data'][1] = $field_types[$field['type']]['label'] . ' ' . t('(module: !module)', array('!module' => $modules[$module_name]->info['name']));
+          $rows[$field_name]['data'][1] = $field_types[$field['type']]['label'];
+          $rows[$field_name]['data'][2] = $modules[$module_name]->info['name'];
         }
 
         // Add the current instance.
         $admin_path = _field_ui_bundle_admin_path($entity_type, $bundle);
-        $rows[$field_name]['data'][2][] = $admin_path ? l($bundles[$entity_type][$bundle]['label'], $admin_path . '/fields') : $bundles[$entity_type][$bundle]['label'];
+        $field_label = $instances[$entity_type][$bundle][$field_name]['label'];
+        $bundle_label = $bundles[$entity_type][$bundle]['label'];
+        if ($admin_path) {
```

Comment:
This is to account for the previous `$rows[$field_name]['data'][2][] = $admin_path ? ...` conditional that would render the bundle name/label as a link to the "Manage fields" page of that bundle, or render it as plain text.

Not sure which cases that was there to cover. Was it permissions (current used not allowed to edit the bundle)? ...or are there cases of bundles/fields (perhaps in contrib if not in core?) that cannot be configured? Anyway, I though that there must have been a reason, so I have retained it. If there is no reason, then we could remove the conditional, and have the field label and bundle label be rendered as links in all cases.